jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/ConfigHelper.java

```diff
@@ -205,6 +206,14 @@ private static void verifyImageNames(List<ImageConfiguration> ret) {
 
         // Initialize configuration and detect minimal API version
         ConfigHelper.initAndValidate(resolvedImages, apiVersion, imageNameFormatter);
+
+        for (ImageConfiguration image : resolvedImages) {
+            BuildConfiguration buildConfiguration = image.getBuildConfiguration();
+            if (buildConfiguration != null &&  buildConfiguration.isDockerFileMode()) {
+                log.info("Using Dockerfile: %s", buildConfiguration.getDockerFile().getAbsolutePath());
+            }
+        }
```

Comment:
We're reiterating through the `resolvedImages` array here.

The [`initAndValidate(List<ImageConfiguration> images, String apiVersion, NameFormatter nameFormatter)` method](https://github.com/eclipse/jkube/blob/1a2898127fdc9b3b30c94d29ee95fb40903a9d8e/jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/ConfigHelper.java#L130-L136) is already doing this.

Even the [`initAndValidate(ConfigHelper.NameFormatter nameFormatter, ImageConfiguration imageConfiguration)`](https://github.com/eclipse/jkube/blob/1a2898127fdc9b3b30c94d29ee95fb40903a9d8e/jkube-kit/build/service/docker/src/main/java/org/eclipse/jkube/kit/build/service/docker/helper/ConfigHelper.java#L239-L249), might be a suitable place.

Do you think any of these other locations might be more suitable for this purpose?

Reply:
OK so maybe it is worth using the java stream api because `init and validate` is one thing, extracting the larger api version is another and logging is again another thing.

Reply:
Well, that method is kind of doing lot's of stuff and the method names really don't help out very much.

It would be much better to refactor, also fix the access modifiers, most of those methods are not used out of that class and shouldn't be public.

Reply:
OK I suggest we merge that one as it is and create an issue for next sprint to refactor that part, how does it sound ?

Reply:
Sounds good, please create the issue and link the PR.

Reply:
https://github.com/eclipse/jkube/issues/1118

Reply:
Thx ;)